src/__tests__/evaluator.test.ts

```diff
@@ -77,4 +85,157 @@ describe('Evaluator', () => {
 
     expect(actual_variation).toEqual('wanted_value');
   });
+
+  test.each([
+    // if (target.attr.email endswith '@harness.io' && target.attr.role = 'developer')
+    ['email_is_dev', 'boolflag_and', 'user@harness.io', 'developer', true],
+    ['email_is_mgr', 'boolflag_and', 'user@harness.io', 'manager', false],
+    ['ext_email_is_dev', 'boolflag_and', 'user@gmail.com', 'developer', false],
+    ['ext_email_is_mgr', 'boolflag_and', 'user@gmail.com', 'manager', false],
+    // if (target.attr.email endswith '@harness.io' || target.attr.email endswith '@somethingelse.com')
+    ['email_is_harness', 'boolflag_or', 'user@harness.io', 'n/a', true],
+    ['email_is_other', 'boolflag_or', 'user@somethingelse.com', 'n/a', true],
+    ['email_is_gmail', 'boolflag_or', 'user@gmail.com', 'n/a', false],
+  ])('Target rules v2 %s_%s', async (name, flagName, email, role, expected) => {
+    async function loadFlags(repo: StorageRepository) {
+      const feature_config_or: FeatureConfig = {
```

Comment:
Try to keep to standard JS naming conventions using camel case instead of snake case
```suggestion
      const featureConfigOr: FeatureConfig = {
```

Reply:
yeah i brought these over from Ruby, I probably left the old names in place. fixed